config/jenkins-jobs_infra.ci.jenkins.io.yaml

```
@@ -219,6 +219,10 @@ jobsDefinition:
             username: "${JIRA_API_INFRA_REPORTS_USERNAME}"
             usernameSecret: true
             password: "${JIRA_API_INFRA_REPORTS_PASSWORD}"
+      mirrorbits-mirrors-list:
+        name: "List of Mirrorbits mirrors"
+        repository: "infra-reports"
+        jenkinsfilePath: "mirrorbits-mirrors-list/Jenkinsfile"
```

WDYT about the following naming which extends the scope from "mirrorbits mirror list" to "Jenkins Infrastructure public data" (as we will be able to add our outbound and inbound IPs to allow user firewall to be set up with automation for example) ?

```suggestion
      jenkins-infra-data:
        name: "Jenkins Infrastructure Public Data"
        repository: "infra-reports"
        jenkinsfilePath: "jenkins-infra-data/Jenkinsfile"
```